truffle/src/main/java/org/truffleruby/options/ByteStringOptionDescription.java

```diff
@@ -41,7 +41,11 @@ public Object checkValue(Object value) {
 
     @Override
     public String toString(Object value) {
-        return new String((byte[]) value, StandardCharsets.US_ASCII);
+        if (value == null) {
+            return "null";
+        } else {
+            return new String((byte[]) value, StandardCharsets.US_ASCII);
```

Comment:
I used US_ASCII because it's the simplest encoding, and we don't actually know the encoding of this byte string. It can depend on the external encoding option. I thought if people are debugging and just want to see these bytes then give them something but we can't be too clever.